### PR TITLE
Added Warning and Info toasts

### PR DESCRIPTION
**Pull request by jfelipebc**
- The Warning toast was added.
- The Info toast was added.

### CODE REVIEW

**Comment by jdjuan** [comment]
@jfelipebc ! Thank you thank you thank you! ❤️ This is our first contribution from the community and we love it!

So much needed! 

I will add a small comment and that's it, after it we can merge! 👍

File: package.json [hunk 1]
```diff
@@ -25,14 +25,17 @@
...
+    "warning",
+    "info"
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/jdjuan/ng-notyf/issues"
   },
   "devDependencies": {
     "@angular/animations": "^4.1.1",
+    "@angular/cli": "^1.1.0",
```
Comment (jdjuan) [inline package.json]: Can you please remove this added dependency. I don't think we are using the CLI with this library 😄 Or am I missing something?

**Review by jdjuan — COMMENTED** [review]
@jfelipebc I just updated your PR with the change! 😄 Hope you don't mind.
I also abtracted the default messages into its own file. What do you think?
@caroso1222 can you approve this please? I already tested the changes locally 👍

**Review by caroso1222 — APPROVED** [review]

**Comment by jfelipebc** [comment]
@jdjuan Amazing!, no problem, the idea to put default messages in separate file is a great idea.

**Comment by jdjuan** [comment]
@jfelipebc thank you again for the contributions, credits will appear in the change log 🎖

**Pull request merged**